Makefile

```diff
@@ -13,10 +13,11 @@ create-instance: ## Creates RDS Instance - mandatory: INSTANCE_NAME=[name];
 	-var 'instance_db_name=$(INSTANCE_NAME)'"
 	echo -e "\n\033[1mRDS Instance Details\033[00m"
 	echo -e "\nEndpoint:"
-	aws rds describe-db-instances --db-instance-identifier=test-rds-instance-nonprod | jq -r '.DBInstances[0].Endpoint.Address'
+	aws rds describe-db-instances --db-instance-identifier=$(INSTANCE_NAME)-nonprod | jq -r '.DBInstances[0].Endpoint.Address'
```

Comment:
Should this be an environment variable or will it always be in non-prod?

Reply:
Good question normal you would want it to configurable, but in this case the RDS instance is only going to be created for testing and development usage, so will only be used in non-prod.